src/java/fr/paris/lutece/plugins/notifygru/modules/forms/services/provider/FormsProvider.java

```diff
@@ -272,13 +274,18 @@ public Collection<InfoMarker> provideMarkerValues( )
         urlFO.addParameter( PARAMETER_ID_FORM_RESPONSES_FO, _nIdFormResponse );
         notifyMarkerFOUrl.setValue( urlFO.getUrl( ) );
         result.add( notifyMarkerFOUrl );
-        
+
+        Locale locale =_request.getLocale();
+        DateFormat dateFormat = DateFormat.getDateInstance(DateFormat.LONG, locale);
+
+        String dateCreateMark =  dateFormat.format(formResponse.getCreation( ));
         InfoMarker creationDateMarker = new InfoMarker( MARK_CREATION_DATE );
-        creationDateMarker.setValue( formResponse.getCreation( ).toString( ) );
+        creationDateMarker.setValue( dateCreateMark.toString( ) );
         result.add( creationDateMarker );
-        
+
+        String dateUpdateMark =  dateFormat.format(formResponse.getUpdate());
         InfoMarker updateDateMarker = new InfoMarker( MARK_UPDATE_DATE );
-        updateDateMarker.setValue( formResponse.getCreation( ).toString( ) );
+        updateDateMarker.setValue( dateUpdateMark.toString( ) );
```

Comment:
Please send a "date" object to the template, the format of the date will be chosen in the template